server/schema/types/root_query_type.js

```diff
@@ -34,23 +34,23 @@ const RootQueryType = new GraphQLObjectType({
             }
         },
         coffee: {
-            type: CoffeeType,
+            type: require("./coffee_type"),
```

Comment:
Why did you change it back to this? Was it not working again? I think the previous syntax looked much cleaner.

Reply:
It was part of the debugging process for errors I was getting with the coffee association. However, changing it back still works so I agree that it's cleaner to revert back

Reply:
@hegelocampus @uwgnol1612  Made both of these changes and ready for your review.